docs/source/verification/verification.rst

```diff
@@ -815,3 +815,82 @@ slight degradation in order-of-accuracy is noted for the thexahedral topology.
+1.0e-5 Pa-s.  *The flow should keep its inflow velocity and temperature profiles throughout the length of the domain*.
+
+The domain is 3000 m long, 1000 m tall, and 20 m wide with 300 x 100 x 2 elements.  The upper and lower boundaries 
+are symmetry with the specified normal gradient of temperature option used such that the gradient matches the initial
+temperature profile with its gradient of 0.01 K/m. Flow enters from the left and exits on the right.  The remaining 
+boundaries are periodic. 
+
+We test the problem on three configurations: 1) using the standard open boundary condition, 
+2) using the global-mass-flow-rate-correction option, and 3) using the standard open boundary condition with a local
+moving-time-averaged reference temperature in the Boussinesq buoyancy term.
+
+Figure :numref:`stratified_outflow_ux1` shows the across-channel profile of outflow streamwise velocity.  It is clear
+that in configuration 1, the velocity is significantly distorted from the correct solution.  Configurations 2 and 3 
+remedy the problem.  However, if we reduce the range of the x-axis, as shown in Figure :numref:`stratified_outflow_ux2`, 
+we see that configuration 3, the use of the standard open boundary condition with a local moving-time-averaged
+Boussinesq reference temperature, provides a superior solution in this case.  In Figure, :numref:`stratified_outflow_T1`,
+we also see that configuration 1 significantly distorts the temperature from the correct solution.
+
+.. _stratified_outflow_ux1:
+
+.. figure:: figures/Ux_123.png
+   :width: 500px
+   :align: center
+
+   Outflow velocity profiles for the thermally stratified slip-channel flow.
+
+
+.. _stratified_outflow_ux2:
+
+.. figure:: figures/Ux_23.png
+   :width: 500px
+   :align: center
+
+   Outflow velocity profiles for the thermally stratified slip-channel flow considering only the case with the
+   global mass-flow-rate correction and the standard open boundary with the local moving-time-averaged Boussinesq
+   reference value.
+
+
+.. _stratified_outflow_T1:
+
+.. figure:: figures/T_123.png
+   :width: 500px
+   :align: center
+
+   Outflow temperature profiles for the thermally stratified slip-channel flow.
+
+
+We also verify that the global mass-flow-rate correction of configuration 2 is correcting the outflow mass flow rate 
+properly.  The output from Nalu showing the correction is as follows:
+
+
+.. code-block:: c++
+ 
+   Mass Balance Review:
+   Density accumulation: 0
+   Integrated inflow:    -188486
+   Integrated open:      188486
+   Total mass closure:   -9.82374e-06
+   A mass correction of: -1.22797e-08 occurred on: 800 boundary integration points:
+   Post-corrected integrated open: 188486
+
+The product of the point-wise mass correction of -1.22797e-06 over all 800 boundary integration points is -9.82374e-06,
+which agrees with the reported pre-correction mass-flow-rate error.
```

Comment:
We may need more precision in the integrated inflow/open output such that mass balance can be verified. The post-corrected could also increate precision for this.

Reply:
Okay, I added precison